SleepTracker/SleepTrackerTests/Helpers/CGFloat+AppFontSizeTests.swift

```diff
@@ -0,0 +1,13 @@
+import XCTest
+
+final class AppFontSizeTests: XCTestCase {
+    func testAppFontSizeReturnsCorrectValues() {
+        let caption: CGFloat = 12
+        let base: CGFloat = 20
+        let heading: CGFloat = 40
+
+        XCTAssertEqual(caption, .AppFontSize.caption)
+        XCTAssertEqual(base, .AppFontSize.base)
+        XCTAssertEqual(heading, .AppFontSize.heading)
```

Comment:
nit:

```suggestion
        XCTAssertEqual(.AppFontSize.caption, 12)
        XCTAssertEqual(.AppFontSize.base, 20)
        XCTAssertEqual( .AppFontSize.heading, 40)
        
```
to be consistent with the `AppUIConstantTests`?